### DISCUSSION TITLE
How to use q-pdfviewer in quasar v2

### DISCUSSION THREAD

**Discussion by blalmal10a** [quasarframework/app-extension-qpdfviewer#118] [Q&A] [answered]
My quasar project doesnt recognize the <q-pdfviewer > component. So i installed this repository and found that @quasar/app is required for the extension to work. But i cannot use the @quasar/app in my quasar v2. Please help me solve this problem. Thank you in advance..

found the error in the image after installing @quasar/app
![quasar](https://user-images.githubusercontent.com/45043442/163571480-3c750d1b-3eeb-4bd1-8974-63f46a36190f.jpg)

**Comment by hawkeye64** [comment, marked as answer]
For Quasar v2/Vue v3, install it this way:

`quasar ext add @quasar/qpdfviewer@next`

Notice the `next` on the end, otherwise, you will get the Quasar v1 compatible version.
